docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -410,13 +410,18 @@ branch `merge-develop-into-master` is correct, and a pull request into
 [Changes LDNS]:                            https://github.com/zonemaster/zonemaster-ldns/blob/develop/Changes
 [Changes Zonemaster]:                      https://github.com/zonemaster/zonemaster-gui/blob/develop/Changes
 [Create Docker Image]:                     ReleaseProcess-create-docker-image.md
+[Declaration of prerequisites]:            ../../../README.md#prerequisites
 [Docker Hub]:                              https://hub.docker.com/u/zonemaster
 [Engine.pm]:                               https://github.com/zonemaster/zonemaster-engine/blob/develop/lib/Zonemaster/Engine.pm
 [Installation.md GUI]:                     https://github.com/zonemaster/zonemaster-gui/blob/develop/docs/Installation.md
 [LDNS.pm]:                                 https://github.com/zonemaster/zonemaster-ldns/blob/develop/lib/Zonemaster/LDNS.pm
+[License string]:                          https://metacpan.org/pod/CPAN::Meta::Spec#license
```

Comment:
It looks like this link is not used in the document.

Reply:
That is correct. And I found a few more. It looks like it was added in this PR, but that is because I changed from small letter L to capital letter L in the beginning of the string. Fixed.